compatibility/bazel_tools/testing.bzl

```diff
@@ -1235,7 +1235,8 @@ def sdk_platform_test(sdk_version, platform_version):
     # can control them individually.
 
     # We allocate parties via @daml/ledger which only supports this since SDK 1.8.0
-    if versions.is_at_least("1.8.0", sdk_version):
+    # Versions prior to 2.0.0 are incompatible with Node version 16.
+    if versions.is_at_least("2.0.0", sdk_version):
```

Comment:
How to proceed with this?
Can we patch or fix the dependency issue, or are these releases incompatible with the more recent NodeJS version?

Reply:
I've kept the older NodeJS version for the compat tests specifically for now. This seems to resolve the issue (at least locally, waiting for CI). This gives us the freedom to look into the NodeJS update for the compat tests independently.